vignettes/site_only/deploy_model.Rmd

```diff
@@ -0,0 +1,189 @@
+---
+title: "Deploying a model"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Getting Started with healthcareai}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include=FALSE}
+set.seed(43170)
+knitr::opts_chunk$set(echo = TRUE, results = "hold", collapse = TRUE,
+                      comment = "#>", eval = FALSE)
+```
+
+Now that you've put in the work of building a model and it's performing well on historical data, congratulations! It's time to think about how to set it up to make reoccurring predictions. At this point, you have
+
+- Decided on an algorithm
+- Decided on your feature (i.e., input variable) list
+- A model that meets performance requirements
+
+# Saving a model
+
+In order to work with models in a production environment, first you must be able to have a saved model on disk. Here's how that's done--pay attention to last line. *Note, install tidyverse via* `install.packages('tidyverse')`
+
+```{r}
+library(healthcareai)
+library(DBI)
+library(tidyverse)
+
+my_con <- build_connection_string(server = "HCS-GM0004",
+                                  database = "SAM")
+
+con <- dbConnect(odbc::odbc(), .connection_string = my_con)
+
+train_query <- "SELECT 
+                 [FacilityAccountID]
+                ,[AdmitAgeNBR]
+                ,[GenderCD]
+                ,[EthnicGroupDSC]
+                ,[MaritalStatusDSC]
+                ,[Readmit30FLG]
+  FROM [SAM].[Schema].[SummaryVisitBASE]
+  WHERE InpatientFLG = 'F'"
+
+d <- db_read(con, train_query)
+
+models <- machine_learn(d = d, 
+                        FacilityAccountID,
+                        models = "rf",
+                        outcome = Readmit30FLG)
+
+# Save my model
+save(models, file = "readmit_models.RDA")
+```
+
+# Pushing predictions to a database
+
+While the frequency will vary depending on the use case, for most healthcare ML scenarios you'll need predictions on a reoccurring basis. Daily batch predictions are common. To create predictions, new rows (representing patients or patient encounters) are run against the model that lives in the RDA file. 
+
+Instead of slimply creating predictions in R, however, in a production scenario you'll be pushing these predictions to a database. Typically we append to the same table each night, such that a person will get an updated risk score for each day they're in the hospital (for example).
+
+## Creating predictions
```

Comment:
I feel like there are three processes that are wrapped up in this section and separating them might make the process clearer to the reader: 1) In dev, check predictions with `glimpse`, figure out how you want to rename columns and add utility columns, 2) Set up the DB to receive the predictions in SAMD or whatever, and 3) have a production script that uses code you wrote in step 1 to push predictions to the DB.

Reply:
Actually, the last line in this section about scheduling the recurring job makes clear this is all a test in dev. Maybe just make that clear at the top of this section.